### PR TITLE
[MIG] porting project_closing to 8.0

### PR DESCRIPTION
**Pull request by charbeljc**
Redoing #95 in three separate PRs, one per ported module.

### CODE REVIEW

File: project_closing/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+##############################################################################
+{
+    "name": "Project closing",
+    "version": "1.1",
+    "author": "Camptocamp,Odoo Community Association (OCA)",
+    "website": "http://www.camptocamp.com",
+    "category": "project Management",
+    "depends": ["project"],
+    "description": """
+    Automatic account analytic closing when related project is closed.
```
Comment (pedrobaeza) [inline project_closing/__openerp__.py]: Move this to README.rst and use OCA's template

File: project_closing/project.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from openerp.osv import orm
+
+
+class project_project(orm.Model):
```
Comment (pedrobaeza) [inline project_closing/project.py]: Use CamelCase

File: project_closing/project.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    _inherit = 'project.project'
+
+    def set_done(self, cr, uid, ids, context=None):
```
Comment (pedrobaeza) [inline project_closing/project.py]: Can you please pass it to new API as it's a little module?

**Comment by vrenaville** [comment]
:+1:

File: project_closing/README.rst [hunk 4]
```diff
@@ -0,0 +1,42 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Project closing
+===============
+
+Automatic account analytic closing when related project is closed.
+and if a projet is opened, the related analytic account will be re-opened.
```
Comment (pedrobaeza) [inline project_closing/README.rst]: s/projet/project

File: project_closing/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Project closing",
+    "version": "1.1",
```
Comment (pedrobaeza) [inline project_closing/__openerp__.py]: You should bump to current version numbering proposal 8.0.1.0

File: project_closing/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
```
Comment (pedrobaeza) [inline project_closing/__openerp__.py]: You can remove this line

File: project_closing/__openerp__.py [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+#    Copyright (C) 2013 Camtpcaomp
```
Comment (pedrobaeza) [inline project_closing/__openerp__.py]: Correct/expand this copyright

File: project_closing/project.py [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+from openerp import api, models
+
+
+class ProjectProject(models.Model):
+    _inherit = 'project.project'
+
+    @api.multi
+    def set_done(self):
+        """ We will close related analytic account """
+
```
Comment (pedrobaeza) [inline project_closing/project.py]: This empty line seems strange after a docstring

Comment (charbeljc) [inline project_closing/project.py]: @pedrobaeza made the change suggested in your comments.

File: project_closing/__openerp__.py [hunk 9]
```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2013 - 2015 Camtpcaomp
```
Comment (pedrobaeza) [inline project_closing/__openerp__.py]: I mean that Camptocamp is not correctly spelled

File: project_closing/__init__.py [hunk 10]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
```
Comment (pedrobaeza) [inline project_closing/__init__.py:4]: You can lighten up this header

File: project_closing/README.rst [hunk 11]
```diff
@@ -0,0 +1,42 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Project closing
+===============
+
+Automatic account analytic closing when related project is closed.
```
Comment (dreispt) [inline project_closing/README.rst]: end line with a comma, not a period

File: project_closing/project.py [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+from openerp import api, models
+
+
+class ProjectProject(models.Model):
+    _inherit = 'project.project'
+
+    @api.multi
+    def set_done(self):
+        """ We will close related analytic account """
+        self.mapped('analytic_account_id').write({'state': 'close'})
```
Comment (dreispt) [inline project_closing/project.py:11]: Smart!

**Comment by dreispt** [comment]
:+1:

File: project_closing/__openerp__.py (hunk 9 above)
Comment (charbeljc) [inline project_closing/__openerp__.py]: Hey Thanks @pedrobaeza you are a very careful reader !

File: project_closing/README.rst (hunk 11 above)
Comment (charbeljc) [inline project_closing/README.rst]: Thank @dreispt typo corrected !

**Comment by pedrobaeza** [comment]
:+1:

**Comment by rafaelbn** [comment]
Very thankful with you opinion here: https://github.com/OCA/project/issues/148

**Pull request merged**